sourcegraph/sourcegraph
# remove redundant sort call on points

Closes https://github.com/sourcegraph/sourcegraph/issues/34250

https://user-images.githubusercontent.com/1855233/171300543-058dbc24-7e4a-42a5-a18c-385d2f028734.mov

## Test plan

- Existing tests and screenshot checks should still pass
- Go to any insight with overlapping data series
- Hover should still render line and points above others

## App preview:

- [Web](https://sg-web-insights-remove-double-sort.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-octasryocz.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
